src/codecs/webp/vp8.rs

```diff
@@ -913,6 +913,28 @@ impl Frame {
         }
     }
 
+    /// Fills an rgba buffer by skipping the alpha values
+    pub(crate) fn fill_rgba(&self, buf: &mut [u8]) {
+        for index in 0..self.ybuf.len() {
```

Comment:
In my opinion, more idiomatic would be

```suggestion
        for (index, rgb_chunk) in (0..self.ybuf.len()).zip(buf.chunks_exact_mut(4)) {
```

Reply:
Yeah I had intended to rewrite some of this, done now.